### DISCUSSION TITLE
/api/batch not found

### DISCUSSION THREAD

**Discussion by Raybrilliant** [pocketbase/pocketbase#6205] [Q&A]
# Issue: Batch API Not Found Error

## Description
I'm encountering an issue when trying to use the batch API to create multiple entries in the `absence` collection. Despite having set the API rules to allow anyone to create and update, I consistently receive a "404 Not Found" error when attempting to send the batch request.

## Code Snippet
Here is the relevant code I'm using:

```javascript
let data = db.createBatch();   
data.collection('absence').create({
    student_id: student_id[0].id,
    status: false,
});
data.collection('absence').create({
    student_id: student_id[1].id,
    status: false,
});

try {
    await data.send();
} catch (error) {
    return fail(400, { message: error.message });
}
```

## Error Message
The error message I receive is:
```
originalError: {
    url: 'http://localhost:55000/api/batch',
    status: 404,
    data: { code: 404, message: 'Not Found.', data: {} }
  }

```

## API Rules
I have verified that the API rules are set to allow anyone to create and update records in the `absence` collection. 

## Steps to Reproduce
1. Set up the database with the `absence` collection.
2. Ensure API rules allow creation and updates.
3. Run the above code to create batch entries.

## Additional Information
- I have checked the collection name and it is correctly spelled as `absence`.
- The `student_id` array contains valid IDs.
- I am using the latest version of the database SDK.

## Questions
- Is there a specific endpoint I need to use for batch operations?
- Are there any additional configurations required for batch processing?
- Could there be any issues with the database setup that might lead to this error?

Any help or guidance on this issue would be greatly appreciated!

**Comment by ganigeorgiev** [comment]
I'm not able to reproduce it (_the above steps are missing what `db` is and how it is initialized but I assume that it is a PocketBase JS SDK instance_).

You are either not using the latest JS SDK v0.22+ and PocketBase executable v0.23+ OR you didn't enable the Batch API from the _Dashboard > Settings_.

If you still cannot identify the culprit, feel free to elaborate what exactly JS SDK and PocketBase executable versions you are using and whether you have enabled the Batch API.
